website/docs/reference/sinks/datadog_logs.md.erb

```diff
@@ -0,0 +1,29 @@
+<%- component = metadata.sinks.datadog_logs -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+## Options
+
+<%= fields(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
+### Obtaining a Sematext token
+
+1. Register for a free account at [datadoghq.com](https://www.datadoghq.com/)
+
+2. Fetch your logs api key by going to the [other](https://app.datadoghq.com/logs/onboarding/other) options
+and selecting the `fluentd` option, it should then present you an `api_key`. This api key can now be used
```

Comment:
Why `fluentd`? Is there not a generic HTTP or curl option?

Reply:
Not that I could find at all and it seemed fluentd/fluentbit had the easier config to parse/read and extract the api key. I think if we want to avoid this we must get vector in there.